web/app/charts/MapChart.js

```diff
@@ -753,18 +756,195 @@ bluewave.charts.MapChart = function(parent, config) {
         if (!opacity) opacity = 1.0;
 
 
+        var curve = d3.line().curve(d3[smoothing]);
```

Comment:
Shouldn't we make all this conditional? Wrap all the bundling stuff in an if statement?
```
if (bundleLines){
```